### DISCUSSION TITLE
I want to make a types-only npm package without a corresponding JS package

### DISCUSSION THREAD

**Discussion by onlywei** [DefinitelyTyped/DefinitelyTyped#69532] [_Misc]
I see that most packages within the DefinitelyTyped repo are types for another package on npm, such as `@types/koa` corresponds to the `koa` package.

I want to make a package containing only types, but there is no corresponding JS package at all. It's just types that I want to share. Should I still attempt to add it to this repo?

If I do add it to this repo, how would people use it? Would they have to do the following?:

```
import { SomeType } from '@types/the-package-i-make';
```

**Comment by Anonymous4078** [comment]
Then where is your package located

**Reply by onlywei** [reply]
There is no package to be located anywhere. I'm just making types.

**Reply by neerajkumar161** [reply]
Then make private packages if you're using only by yourself. You can publish types using private packages. Maybe there are other ways, like making public packages as well. It depends on you.
